keyboards/kprepublic/bm16v2/config.h

```diff
@@ -0,0 +1,7 @@
+// Copyright 2022 fauxpark (@fauxpark)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define RGB_DI_PIN B9
```

Comment:
This pin should support `WS2812_DRIVER = pwm` using the `TIM4_CH4` function.

Reply:
Done, not sure if the config is entirely correct but it works.